shared/util/typed-put.js.flow

```diff
@@ -1,25 +1,12 @@
 // @flow
-// import type {PutEffect, Channel} from 'redux-saga'
-// import type {TypedActions} from '../actions/typed-actions-gen'
-// declare var put: {
-// (action: TypedActions): PutEffect<TypedActions, null>,
-// (channel: Channel, action: TypedActions): PutEffect<TypedActions, Channel>,
-// resolve: {
-// (action: TypedActions): PutEffect<TypedActions, null>,
-// (channel: Channel, action: TypedActions): PutEffect<TypedActions, Channel>,
-// },
-// }
-
-// Uncomment the above to have a typed put. Current flow makes this VERY VERY slow. The merge phase timesout on several files.  This was super useful to find thunks but is likely not that useful now
```

Comment:
turns out flow works with this now. in my testing a flow check runs 2 seconds slower w/ this and increases coverage a lot